mapper.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 var (
-	metricLineRE      = regexp.MustCompile(`^(\*\.|[^*.]+\.)*(\*|[^*.]+)$`)
+	metricLineRE      = regexp.MustCompile(`^(\*\.|[^*.]+\.|\.)*(\*|[^*.]+)$`)
```

Comment:
I've not found any mention of group names or types in the graphite data format. Do you have a reference?